third_party/contour-head/gateway/gateway-internal.yaml

```diff
@@ -65,10 +70,6 @@ spec:
         from: "All"
   - protocol: HTTPS
     port: 443
-    tls:
-      mode: Terminate
-      routeOverride:
-        certificate: Allow
```

Comment:
Are these changes related?

Reply:
It is not related. I reverted it to be clear.